ops/systemd/export_workers.env

```diff
@@ -1,16 +1,23 @@
 ## General settings
-HOSTNAME=https://export.hotosm.org
+HOSTNAME=export-staging.hotosm.org
```

Comment:
The strings are example strings. Please make 

`HOSTNAME=export-tool.example.org`  

I am assuming from your diff that the `https://` scheme must not be specified